src/__tests__/lib/command/util/schema-util.test.ts

```diff
@@ -11,9 +11,10 @@ import type {
 } from '@smartthings/core-sdk'
 
 import type { Profile } from '../../../../lib/cli-config.js'
+import type { stdinIsTTY, stdoutIsTTY } from '../../../../lib/io-util.js'
 import type { clipToMaximum, fatalError } from '../../../../lib/util.js'
 import type { APICommand } from '../../../../lib/command/api-command.js'
-import type { chooseOrganization, organizationDef } from '../../../../lib/command/util/organizations-util.js'
+import { chooseOrganization, organizationDef } from '../../../../lib/command/util/organizations-util.js'
```

Comment:
Here and on line 14 (`import type { stdinIsTTY, stdoutIsTTY }...`) the import statement isn't what I expected with regard to `type` and not `type`

Reply:
Yeah, I missed `type` on a bunch of things here. Fixed now.